client/src/pages/Admin/HowItWorks/index.js

```diff
@@ -0,0 +1,69 @@
+import Title from '../../../components/Title';
+import * as T from '../../../components/Typography';
+import Button from '../../../components/Button';
+import TextWithIcon from '../../../components/TextWithIcon';
+import { Row, Col } from '../../../components/Grid';
+import { navRoutes } from '../../../constants';
+import * as S from './style';
+const HowItWorks = () => {
+  return (
+    <S.Wrapper>
+      <Title lightSection="How it " boldSection="works" mbM="3" />
+      <Row mb="6" mbM="4">
+        <Col w={[4, 12, 12]}>
+          <T.P color="gray8">
+            On this page you can find useful links to user guide(s) and videos
+            to make sure you are making the most of this tool for your
+            therapists and their clients.
+          </T.P>
+        </Col>
+      </Row>
+
+      <Row mb="6" mbM="4">
+        <Col w={[4, 12, 12]}>
+          <Col w={[4, 12, 12]}>
+            <TextWithIcon
+              to="https://calendar.google.com/"
+              external="true"
+              underline="true"
+              text="View video"
+              icon="forwardArrow"
+              weight="bold"
+            />
+          </Col>
+        </Col>
+      </Row>
+
+      <Row mb="6" mbM="4">
+        <Col w={[4, 12, 12]}>
+          <Col w={[4, 12, 12]}>
+            <TextWithIcon
+              to="https://calendar.google.com/"
+              external="true"
+              underline="true"
+              text="Read our user guide(s)"
+              icon="forwardArrow"
+              weight="bold"
+            />
+          </Col>
+        </Col>
+      </Row>
+      <Row mb="6" mbM="4">
+        <Col w={[4, 12, 12]}>
+          <T.H3 color="gray8" weight="bold">
+            Still need help?
+          </T.H3>
+          <T.P color="gray8">
+            Contact the Duet team at digital@duetplatform.com
+          </T.P>
+        </Col>
+      </Row>
+      <S.ButtonWrapper>
+        <Col w={[4, 8, 8]}>
+          <Button to={navRoutes.ADMIN.SIGNUP} text="Return home" />
+        </Col>
+      </S.ButtonWrapper>
```

Comment:
no need to ButtonWrapper you can add flex = 1 on Col style that would be enough
```suggestion
        <Col w={[4, 8, 8]} style={{flex: 1}}>
          <Button to={navRoutes.ADMIN.SIGNUP} text="Return home" />
        </Col>
```